### DISCUSSION TITLE
Report issues my apk

### DISCUSSION THREAD

**Discussion by Lawamenahaulala23** [termux/termux-packages#13353] [General]
can you help me with related matters in my application regarding report issues?

**Comment by ChiefMikeK** [comment]
This is a very vague question as **your** [My repo](https://github.com/Lawamenahaulala23/Lawamenahaulala23/pulsel) is empty.

- What are you asking?
- Reddit is a better place to search for answers / ask for help.
- Please be specific when asking for help!

**Comment by sylirre** [comment]
Depends on mean of "my application". If you are actually about your Termux installation, please write full description about faced issue. With screenshots, if possible.

However if you went there to ask help with development of your own application, please seek for help elsewhere. Termux GitHub community is only about Termux (should be obviously) and its development.
